### DISCUSSION TITLE
Very cool!

### DISCUSSION THREAD

**Discussion by TysonP2** [KeithRochester/Standard-Monitoring-Framework#1] [General]
Very cool solution. SCOM is so powerful but at the same time it's incredibly complex. I appreciate any efforts that reduce the complexity of SCOM. Fantastic SCOMathon Coffee Break [presentation](https://www.youtube.com/watch?v=yEjRj8WPNJ8), BTW. Thank you for sharing!
-Tyson

**Comment by KeithRochester** [comment]
Thanks @TysonP2. The idea came from your URLGenie MP. I love the principle of people configuring not authoring monitoring.
